Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -481,6 +481,34 @@ Messages::WebDriverClient::CloseWindowResponse WebDriverConnection::close_window
     return get_window_handles().take_response();
 }
 
+// 11.3 Switch to Window, https://w3c.github.io/webdriver/#dfn-switch-to-window
+Messages::WebDriverClient::SwitchToWindowResponse WebDriverConnection::switch_to_window(const AK::JsonValue& payload)
+{
+    // 1. Let handle be the result of getting the property "handle" from the parameters argument.
+    // 2. If handle is undefined, return error with error code invalid argument.
+    auto const& handle = TRY(get_property(payload, "handle"sv));
+
+    // FIXME: 3. If there is an active user prompt, that prevents the focusing of another top-level browsing
+    //           context, return error with error code unexpected alert open.
```

Comment:
The way our dialogs are implemented, we can't switch tabs if a dialog is open. So we might as well implement this step and return:
```c++
if (m_page_client.page().has_pending_dialog())
    return Web::WebDriver::Error::from_code(Web::WebDriver::ErrorCode::UnexpectedAlertOpen, "A user dialog is open"sv);

```